src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -1337,7 +1339,7 @@ public void Incremental_Generators_Exception_During_Execution()
             var source = @"
 class C { }
 ";
-            var parseOptions = TestOptions.Regular.WithLanguageVersion(LanguageVersion.Preview);
+            var parseOptions = TestOptions.RegularPreview;
```

Comment:
>RegularPreview [](http://example.com/codeflow?start=43&length=14)

`Regular10` here and below?

Reply:
The `GeneratorDriver` only uses incremental general when LangVer==Preview at the moment. That is kept that way intentionally (based on chat with Chris).